src/index.js

```diff
@@ -0,0 +1,147 @@
+// --------------------------------wave3 and 6 Naming the City----------------------
+const cityNameLabel = document.getElementById('cityname');
+const cityNameInput = document.getElementById('cityname-input');
+const resetBtn = document.getElementById('reset-btn');
+// change city name on top header
+const addCityName = document.getElementById('topCityName');
+var currCity;
+// handle Get Realtime Temperature button to check weather
+const gettemperature = document.getElementById('gettemperature');
+var count;
+
+cityNameInput.addEventListener('change', (event) => {
+  currCity = event.target.value;
+  console.log('change:', currCity);
+  cityNameLabel.innerText = currCity;
+  addCityName.innerText = currCity;
+});
+resetBtn.addEventListener('click', () => {
+  cityNameInput.value = '';
+  cityNameLabel.innerText = 'City Name';
+});
+
+gettemperature.addEventListener('click', () => {
+  findLocation(currCity);
+});
+// --------------------------------------wave4 Calling APIs-------------------
+const findLocation = (query) => {
```

Comment:
I like how you take the query in as an argument! However, this function is misleadingly named. It doesn't only find the location - it also finds the temperature at the location and updates your elements with the information.